proposals/migrate-cli.md

```diff
@@ -0,0 +1,68 @@
+# Migrate Bigchaindb cli for Tendermint
+
+## Problem Description
+With Tendermint integration some of the cli sub-commands have been rendered obsolete. It would be only appropriate to white list sub-commands depending on the backend.
+
+### Use cases
+- Avoid confusing the user by not displaying irrelevant sub-commands.
+
+
+## Proposed Change
+Following sub-commands should be updated
+
+- `bigchaindb --help`: list the relevant sub-commands depending on the configured backend. In case the backend is not configured then the default backend `mongodb` should be assumed.
```

Comment:
In BigchainDB-with-Tendermint, I thought the only supported backend was `localmongodb`.

Reply:
The [comment](https://github.com/bigchaindb/bigchaindb/pull/1976#discussion_r158327626) should clarify this.